std_msgs/Float64 ang0
std_msgs/Float64 ang1
std_msgs/Float64 ang2
std_msgs/Float64 ang3
std_msgs/Float64 ang4
std_msgs/Float64 ang5

std_msgs/Float64 vel0
std_msgs/Float64 vel1
std_msgs/Float64 vel2
std_msgs/Float64 vel3
std_msgs/Float64 vel4
std_msgs/Float64 vel5

joint_angles robot_angles
joint_vel robot_velocities

